src/main/java/CourseCreator.java

```diff
@@ -3,95 +3,86 @@
 import java.time.DayOfWeek;
 import java.time.LocalTime;
 import java.util.ArrayList;
-import java.util.List;
 import java.util.Map;
-import java.util.Objects;
-import java.util.stream.Collectors;
 
 /** This class represents a Course Creator. This class uses APIWorker to generate Course objects. */
 public class CourseCreator {
 
     private static final Map<String, DayOfWeek> toDay =
             Map.of(
-                    "MO", DayOfWeek.of(1),
-                    "TU", DayOfWeek.of(2),
-                    "WE", DayOfWeek.of(3),
-                    "TH", DayOfWeek.of(4),
-                    "FR", DayOfWeek.of(5));
+                    "MO",
+                    DayOfWeek.of(1),
+                    "TU",
+                    DayOfWeek.of(2),
+                    "WE",
+                    DayOfWeek.of(3),
+                    "TH",
+                    DayOfWeek.of(4),
+                    "FR",
+                    DayOfWeek.of(5));
 
     /**
-     * Generates a course given a courseId.
+     * Generates a course given a courseID and the session of the course
      *
-     * @param courseId the id of the course to be created.
+     * @param courseId the identifier for hte course
+     * @param session the semester that the course takes place ie S, F, Y
      * @return a Course object
+     * @throws IOException i dont know what exception this throws lol
```

Comment:
If the API doesn't work it should throw IO exception, so you use it any time you create a API Woker